### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
-Add contact form where users can give feedback and provide emails.
- Add list of icons at the bottom of the website.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -409,3 +419,66 @@ li {
   line-height: 43px;
   color: white;
 }
+
+.contactMe {
+  width: 375px;
+  padding-top: 30px;
+}
```
Comment (Wes-Isaac) [inline style.css:426]: - [ ] Great job 👍. It's better to layout this part of the form using flexbox, as it is one of the requirements.
- [ ] The fixed width is limiting the flexibility of the site and creating unnecessary space, try it without setting a width or use more flexible width like `rem`, `em` or `%`.

Your Implementation | Figma Design
:---: | :---:
image | image

Kindly fix this.

File: style.css [hunk 2]
```diff
@@ -409,3 +419,66 @@ li {
...
+
+.interest {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
```
Comment (Wes-Isaac) [inline style.css:435]: - [ ] Great job so far. However, there is a slight difference between your design and the Figma's for this header, ( font, color, spacing).

Your Implementation | Figma Design
:---: | :---:
image | image


Kindly fix this.

File: style.css [hunk 3]
```diff
@@ -409,3 +419,66 @@ li {
...
+
+.inTouch {
+  background-color: #ff6b00;
+  width: 150px;
+  height: 48px;
+  left: 98px;
+  top: calc(50% - 48px / 2 + 141px);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
```
Comment (Wes-Isaac) [inline style.css:465]: - [ ] Good job styling the button but there is a slight difference between your design and the Figma's for this button.
Your Implementation | Figma Design
:---: | :---:
image  image

Kindly fix this.

- [ ] Please add button interactions such as hover, active, disabled and enable, as it is one of the requirements.

image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi Team, 

Impressive work so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

### 📝👌 Your project highlights 🔥😎

- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌
- Awesome design 🙌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -240,5 +240,82 @@ <h2 class="resumeInfo">GET MY RESUME</h2>
         </ul>
       </div>
     </section>
+    <footer class="contactMe">
+      <h2 class="interest">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
```
Comment (Deepakdanger) [inline index.html:247]: - [ ]  [**OPTIONAL**] Kindly adjust the font-weight of this line to match the original website.

**Review by Deepakdanger — APPROVED** [review]
Hi @MICHAELMUNAVU83, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] _Check the comment below_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**